docs/source/load-balancing/load-balancing.md

```diff
@@ -84,15 +86,15 @@ first element of the load balancing plan is needed, so it's usually unnecessary
 to compute entire load balancing plan. To optimize this common case, the
 `LoadBalancingPolicy` trait provides two methods: `pick` and `fallback`.
 
-`pick` returns the first node to contact for a given query, which is usually
+`pick` returns the first element to contact for a given query, which is usually
 the best based on a particular load balancing policy. If `pick` returns `None`,
 then `fallback` will not be called.
```

Comment:
This is no longer true, see 88f46058ca9db98df0e54809127a925da926ec5d. If you can, please fix my oversight. The correct behaviour of `pick/fallback` is described in the `LoadBalancingPolicy`'s docstring.

Reply:
Done